config/ccms/attribute_block_configs/non_passported.yml.orig

```diff
@@ -0,0 +1,2000 @@
+    generate_block?: true
+    value: false
+    br100_meaning: 'Unknown'
+    response_type: boolean
+    user_defined: false
+  NON_MAND_EVIDENCE_OMBUD_RPT:
+    generate_block?: true
+    value: false
+    br100_meaning: 'Unknown'
+    response_type: boolean
+    user_defined: false
+  NON_MAND_EVIDENCE_PLEADINGS:
+    generate_block?: true
+    value: false
+    br100_meaning: 'Unknown'
+    response_type: boolean
+    user_defined: false
+  NON_MAND_EVIDENCE_PRE_ACT_DISC:
+    generate_block?: true
+    value: false
+    br100_meaning: 'Unknown'
+    response_type: boolean
+    user_defined: false
+  NON_MAND_EVIDENCE_SEP_STATE:
+    generate_block?: true
+    value: false
+    br100_meaning: 'Unknown'
+    response_type: boolean
+    user_defined: false
+  NON_MAND_EVIDENCE_WARNING_LTTR:
+    generate_block?: true
+    value: false
+    br100_meaning: 'Unknown'
+    response_type: boolean
+    user_defined: false
+  PREP_OF_STATEMENT_PAPERS:
+    generate_block?: true
+    value: false
+  PROCS_INCLUDE_CHILD:
+    generate_block?: true
+    value: false
+  PROSCRIBED_ORG_APPEAL_COMM:
+    generate_block?: true
+    value: false
+  PUB_AUTH_QUESTION_APPLIES:
+    generate_block?: true
+    value: false
+  PUBLIC_LAW_NON_FAMILY:
+    generate_block?: true
+    value: false
+  REQUESTED_COST_LIMIT_OVER_25K:
+    generate_block?: true
+    value: false
+  RISK_SCA_PR:
+    generate_block?: true
+    value: false
+  ROUTING_COMPLEX_MERITS:
+    generate_block?: true
+    value: false
+  ROUTING_IMMIGRATION:
+    generate_block?: true
+    value: false
+  ROUTING_MENTAL_HEALTH:
+    generate_block?: true
+    value: false
+  ROUTING_SCU:
+    generate_block?: true
+    value: false
+  ROUTING_VHCC:
+    generate_block?: true
+    value: false
+  SCA_APPEAL_INCLUDED:
+    generate_block?: true
+    value: false
+  SCA_AUTO_GRANT:
+    generate_block?: true
+    value: false
+  SMOD_APPLICABLE_TO_MATTER:
+    generate_block?: true
+    value: false
+  SPECIAL_CHILDREN_ACT_APP:
+    generate_block?: true
+    value: false
+  SPECIAL_IMM_APPEAL_COMMISSION:
+    generate_block?: true
+    value: false
+  SUPREME_COURT:
+    generate_block?: true
+    value: false
+  UPPER_TRIBUNAL_IMM_ASY:
+    generate_block?: true
+    value: false
+  UPPER_TRIBUNAL_MENTAL_HEALTH:
+    generate_block?: true
+    value: false
+  UPPER_TRIBUNAL_OTHER:
+    generate_block?: true
+    value: false
+  URGENT_APPLICATION:
+    generate_block?: true
+    value: false
+  URGENT_DIRECTIONS:
+    generate_block?: true
+    value: false
+  MEDIATION_APPLICABLE:
+    generate_block?: true
+  MENTAL_HEAL_QUESTION_APPLIES:
+    generate_block?: true
+  MERITS_SUBMISSION_PAGE:
+    generate_block?: true
+  NEW_APPLICATION:
+    generate_block?: true
+  PROPORTIONALITY_QUESTION_APP:
+    generate_block?: true
+  PROVIDER_HAS_DP:
+    generate_block?: true
+  ROUTING_STD_FAMILY_MERITS:
+    generate_block?: true
+  SA_INTRODUCTION:
+    generate_block?: true
+<<<<<<< HEAD
```

Comment:
Merge conflict remant

Reply:
Oh, .orig files left in by mistake.  removed now.